### PR TITLE
Add config provider tests

### PR DESCRIPTION
**Pull request by farshidtz**

### CODE REVIEW

File: test/suites/edgex-config-provider/main_test.go [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+)
+
+const (
+	platformSnap = "edgexfoundry"
+	provider     = "edgex-config-provider-example"
+)
+
+var services = []string{
+	"device-virtual",
+	/////////////////
```
Comment (MonicaisHer) [inline test/suites/edgex-config-provider/main_test.go:18]: is this note?

File: .github/workflows/config-provider-testing.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      #   with:
+      #     repository: canonical/edgex-config-provider
+      #     ref: snap-testing
+
+      # - name: Setup snapcraft
+      #   uses: snapcore/action-build@v1
+      #   id: snapcraft
+      
+      # - name: Checkout the local actions
+      #   uses: actions/checkout@v3
```
Comment (MonicaisHer) [inline .github/workflows/config-provider-testing.yml:28]: are these notes?

File: test/suites/edgex-config-provider/main_test.go (hunk 1 above)
Comment (farshidtz) [reply test/suites/edgex-config-provider/main_test.go]: It was to separate device virtual from the remaining alphabetically sorted items. I removed and sorted all.

**Review by MonicaisHer — APPROVED** [review]
@farshidtz Thank you for the changes, looks great!

**Pull request merged**